client/state/stats/lists/utils.js

```diff
@@ -189,8 +189,14 @@ export function getChartLabels( unit, date, localizedDate ) {
 		const isWeekend = 'day' === unit && ( 6 === dayOfWeek || 0 === dayOfWeek );
 		const labelName = `label${ unit.charAt( 0 ).toUpperCase() + unit.slice( 1 ) }`;
 		const formats = {
-			day: 'MMM D',
-			week: 'MMM D',
+			day: translate( 'MMM D', {
```

Comment:
:disappointed: 25 existing translations will be lost with this change.

Reply:
Bot, you're wrong. I am adding a `translate()` call, not removing one.